src/olympia/devhub/tests/test_views_submit.py

```diff
@@ -1320,7 +1320,7 @@ def test_finish_submitting_listed_static_theme(self):
         # First link is to edit listing.
         assert links[0].attrib['href'] == self.addon.get_dev_url()
         # Second link is back to my submissions.
-        assert links[1].attrib['href'] == reverse('devhub.addons')
+        assert links[1].attrib['href'] == reverse('devhub.themes')
```

Comment:
Ah!  I thought it was strange that only one test failed - you've identified a bug in our tests - `test_finish_submitting_unlisted_static_theme` should have failed too.  (`self.test_finish_submitting_unlisted_addon` is missing `()`)

Easiest fix is to replace the lines `# Page should be identical to extensions` and `self.test_finish_submitting_unlisted_addon` with a copy of the contents of the `test_finish_submitting_unlisted_addon` test, and change the last `reverse` like you've done above.

Reply:
@eviljeff, did exactly as you said was the easiest way ;)
Is that good?